gcc/rust/lint/rust-lint-marklive.cc

```diff
@@ -113,12 +114,14 @@ MarkLive::visit (HIR::MethodCallExpr &expr)
 
   // node back to HIR
   HirId ref;
-  node_id_to_hir_id (expr.get_mappings ().get_crate_num (), ref_node_id, ref,
-		     expr.get_locus ());
+  bool ret
```

Comment:
usually we do bool ok = mappings->lookup_node_to_hir

I think 'ok' reads better

Reply:
Yeah, sure it is